core/src/main/scala/spire/math/Number.scala

```diff
@@ -525,7 +525,7 @@ private[math] case class DecimalNumber(n: BigDecimal) extends Number { lhs =>
 
   def floor: Number = Number(n.floor)
   def ceil: Number = Number(n.ceil)
-  def round: Number = Number(n.round())
+  def round: Number = ??? //Number(n.round)
```

Comment:
Somehow I missed this but I have a fix now. Again there is an issue with the change of implicits selection algorithm.
The fix I'm doing is:
`  def round: Number = Number(isRealOps(n).round)`